.vscode/tasks.json

```diff
@@ -36,19 +36,6 @@
         "kind": "build",
         "isDefault": true
       }
-    },
```

Comment:
Yeah, Davy and I discussed and this isn't how we debug, and also it doesn't work so it misleads you if you try.

Reply:
Don't think it's needed. It is a bit awkward since we squash-merge so you lose the exact history, but the message is at least folded into the squashed commit.